src/Nethermind/Nethermind.Runner/configs/denver.cfg

```diff
@@ -0,0 +1,47 @@
+{
+  "Init": {
+    "GenesisHash": "0x203e164cf3b6f6765abf2f0355a4d09e5eaf6a777243bc4407b569431cd95cb3",
+    "ChainSpecPath": "chainspec/denver.json",
+    "MemoryHint": 768000000
+  },
+  "Sync": {
+    "FastSync": true,
+    "FastBlocks": true,
+    "UseGethLimitsInFastBlocks": false,
+    "FastSyncCatchUpHeightDelta": 10000000000
+  },
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "AdditionalRpcUrls": [
+      "http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client"
+    ]
+  },
+  "AuRaMerge": {
+    "Enabled": true
+  },
+  "Aura": {
+    "TxPriorityContractAddress": "0x4100000000000000000000000000000000000000",
```

Comment:
Why has this contract to be specified here?

Reply:
I copied it from [here](https://github.com/gnosischain/consensus-deployment-ansible/blob/043171e6cdf3e55e55c97b8f3933e3d1d7d6bc57/denver/custom_config_data/configs/nethermind_config.cfg#L16). is it not being used or smth?

Reply:
You tell me, all I know is that AuRa has many different contracts but only this one is specified here. I don't know the internals of Nethermind, nor which contracts have to be specified here and for what reason

Reply:
So priorities contract is used for potentially sorting transactions when creating blocks. Priorities can come from contract or local configuration. Priorities are optional. Not sure if you deployed this contract on denver. More information can be found here: https://github.com/NethermindEth/nethermind/issues/2300

Reply:
Also. I see [here](https://github.com/gnosischain/consensus-deployment-ansible/blob/6357cf13b841c4420d6f691d19ce6629cd944a82/denver/custom_config_data/nethermind_genesis.json#L288-L291) that you have a contract deployed to this address in denver genesis. That's what is being used as `TxPriorityContract` in denver.

Reply:
Got it, thanks!